testci/test_slice.py

```diff
@@ -0,0 +1,71 @@
+import os
+import csv
+import shutil
+import pytest
+import tempfile
+import pelops.datasets.slice as slice
+
+
+@pytest.fixture
+def slice_env():
+    """Setup mock STR SLiCE dataset"""
+    work_dir = tempfile.mkdtemp('_pelops_testing')
```

Comment:
Prefer pytest's built in `tmpdir` fixtures: https://docs.pytest.org/en/latest/tmpdir.html